src/handlers/localstorage/uniswap.ts

```diff
@@ -10,18 +18,31 @@ const ASSETS = 'uniswapassets';
 const LIQUIDITY = 'uniswapliquidity';
 const UNISWAP_POSITIONS = 'uniswapPositions';
 const UNISWAP_FAVORITES = 'uniswapFavorites';
+const UNISWAP_FAVORITES_METADATA = 'uniswapFavoritesMetadata';
 const uniswapLiquidityVersion = '0.2.0';
 const uniswapPositionsVersion = '0.1.0';
 
 export const uniswapAccountLocalKeys = [ASSETS, LIQUIDITY, UNISWAP_POSITIONS];
 
-export const getUniswapFavorites = (network: any) =>
+export const getUniswapFavorites = (network: any): Promise<EthereumAddress[]> =>
   // @ts-expect-error ts-migrate(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
   getGlobal(UNISWAP_FAVORITES, DefaultUniswapFavorites[network]);
 
 export const saveUniswapFavorites = (favorites: any) =>
   saveGlobal(UNISWAP_FAVORITES, favorites);
 
+export const getUniswapFavoritesMetadata = (
+  network?: string
```

Comment:
can use `Network` type instead of `string`

Reply:
can also default it to `Network.mainnet` as `network = Network.mainnet` and remove the `|| 'mainnet'` on line 39